tensorboard/webapp/core/effects/core_effects.ts

```diff
@@ -84,6 +87,33 @@ export class CoreEffects {
     {dispatch: false}
   );
 
+  /**
+   * HACK: COMPOSITE ACTION -- rationale: most plugins want to be able to tell
```

Comment:
Would be nice to first describe what this effect is trying to accomplish.  I think it's something along the lines of "Dispatches changePlugin action when first active plugin is loaded after page load."